netplugin/agent/docker_event.go

```diff
@@ -28,64 +28,80 @@ import (
 	"golang.org/x/net/context"
 )
 
-// Handles docker events monitored by dockerclient. Currently we only handle
-// container start and die event*/
+// Handles docker events monitored by dockerclient. Currently we only handle // container start and die event*/
 func handleDockerEvents(event *dockerclient.Event, ec chan error, args ...interface{}) {
 
 	log.Debugf("Received Docker event: {%#v}\n", *event)
 	endpointUpdReq := &master.UpdateEndpointRequest{}
-	switch event.Status {
-	case "start":
+
+	var containerID string
+	var networkID string
+
+	if (event.Type == "container" && event.Action == "start") ||
```

Comment:
Hi Divya ... sorry, missed replying to this comment earlier. I did look through the docker api (and specially the sam alba code which we use to interface with the docker api). There are no standard docker enums defined and the code just works with json objects (verified with Ranjith as well) so I left the strings as they are currently ... thanks!